src/packages/galaxy/galaxy.py

```diff
@@ -60,6 +61,15 @@ class Galaxy:
     }
     MAX_PLOT_DISTANCE = 20000
 
+    MAX_RETRIES = 3
+    "The maximum number of times to retry a failed HTTP request before failing permanently."
+
+    TIMEOUT = ClientTimeout(total=10)
+    "A ClientTimeout object representing the total time an HTTP request can take before failing."
+
+    RETRY_SLEEP = 1
```

Comment:
[style] please rename this to `RETRY_DELAY`, as it is a tad more descriptive of what this constant does, and is less likely to be confused with `time.sleep`

Reply:
Variable was removed entirely in lieu of a helper method.